lib/plugins/interactiveCli/setupAws.js

```diff
@@ -39,76 +39,85 @@ const awsSecretAccessKeyInput = () =>
     })
     .then(({ secretAccessKey }) => secretAccessKey.trim());
 
+const steps = {
+  writeOnSetupSkip: () =>
+    process.stdout.write(`\nYou can setup your AWS account later. More details available here:
+
+  http://slss.io/aws-creds-setup\n`),
+  shouldSetupAwsCredentials: () =>
+    confirm('Do you want to set them up now?', { name: 'shouldSetupAwsCredentials' }).then(
+      isConfirmed => {
+        if (!isConfirmed) steps.writeOnSetupSkip();
+        return isConfirmed;
+      }
+    ),
+  ensureAwsAccount: () =>
+    confirm('Do you have an AWS account?', { name: 'hasAwsAccount' }).then(hasAccount => {
+      if (!hasAccount) {
+        return openBrowser('https://portal.aws.amazon.com/billing/signup').then(() =>
+          inquirer.prompt({
+            message: 'Press Enter to continue after creating an AWS account',
+            name: 'createAwsAccountPrompt',
+          })
+        );
+      }
+      return null;
+    }),
+  ensureAwsCredentials: serverless => {
+    const region = serverless.getProvider('aws').getRegion();
+    return openBrowser(
+      `https://console.aws.amazon.com/iam/home?region=${region}#/users$new?step=final&accessKey&userNames=serverless&permissionType=policies&policies=arn:aws:iam::aws:policy%2FAdministratorAccess`
+    ).then(() =>
+      inquirer.prompt({
+        message: 'Press Enter to continue after creating an AWS user with access keys',
+        name: 'generateAwsCredsPrompt',
+      })
+    );
+  },
+  inputAwsCredentials: () => {
+    return awsAccessKeyIdInput().then(accessKeyId =>
+      awsSecretAccessKeyInput().then(secretAccessKey =>
+        awsCredentials
+          .saveFileProfiles(new Map([['default', { accessKeyId, secretAccessKey }]]))
+          .then(() =>
+            process.stdout.write(
+              `\n${chalk.green(
+                `AWS credentials saved on your machine at ${chalk.bold(
+                  '~/.aws/credentials'
```

Comment:
This is more like a stylistic issue (and we have the same code somewhere else in our codebase) but Windows users might get confused by this.

Reply:
Good point. I'll probably prepare another PR, that improves that (in context of this PR, this code was just moved, but not changed)